downstream/modules/platform/ref-controller-database-settings.adoc

```diff
@@ -1,68 +1,42 @@
 [id="ref-controller-database-settings"]
 
-= Database Settings
+= PostgreSQL database configuration and maintenance for {ControllerName}
 
-Configure the following settings in the database to improve performance:
+To improve the performance of {ControllerName}, you can configure the following configuration parameters in the database:
 
-* *Autovacuuming*. Setting this PostgreSQL setting to `true` is a good practice. 
-However, autovacuuming does not occur if there is never any idle time on the database. 
-If you see that autovacuuming is not sufficiently cleaning up space on the database disk, then scheduling specific vacuum tasks during specific maintenance windows can provide a solution.
-* *GUC* parameters. 
-The following are _Grand Unified Configuration_ (GUC) parameters recommended for memory management in PostgreSQL, which are helpful for improving the performance of the database server. 
-Recommended settings for each parameter are also provided:
 
-* `shared_buffers` (integer)
-* `work_mem` (integer)
-* `maintenance_work_mem` (integer)
+*Maintenance* 
 
-These parameters are found in the `postgresql.conf` file, inside the `$PDATA` directory, which manages the configurations of the database server.
+Important maintenance activities that can impact performance are the `VACUUM` and `ANALYZE` tasks. In normal PostgreSQL operation, tuples that are deleted or obsoleted by an update are not physically removed from their table; they remain present until a `VACUUM` is done. Therefore it's necessary to do VACUUM periodically, especially on frequently-updated tables. `ANALYZE` collects statistics about the contents of tables in the database, and stores the results in the `pg_statistic` system catalog. Subsequently, the query planner uses these statistics to help determine the most efficient execution plans for queries. Autovacuum automates the execution of `VACUUM` and `ANALYZE` commands. Setting this PostgreSQL configuration parameter to *true* is a good practice. However, autovacuuming will not occur if there is never any idle time on the database. If it is observed that autovacuuming is not sufficiently cleaning up space on the database disk, then scheduling specific vacuum tasks during specific maintenance windows can be a solution. 
```

Comment:
You mention VACUUM here but the description of autovacuuming has been removed. Where do you explain what 'VACUUM' is?